packages/mail/express/render.js

```diff
@@ -0,0 +1,16 @@
+const { getTemplate } = require('../lib/sendMailTemplate')
+
+module.exports = async (server, pgdb, t) => {
+  server.get(
+    '/mail/render/:template',
+    async (req, res) => {
+      const html = await getTemplate(req.params.template)
+
+      if (!html) {
+        return res.sendStatus(404)
+      }
+
+      return res.send(html)
+    }
+  )
+}
```

Comment:
what's the purpose of this public endpoint?

Reply:
Soley for testing and debugging purposes.

Reply:
ok with me